.gitignore

```diff
@@ -94,5 +94,16 @@ ENV/
 # SQLite3
 db.sqlite3
 
+# vscode
+.vscode
 
-.vscode
\ No newline at end of file
+# vim temporary files
+.*.swp
+.*.swn
+.*.swo
+
+# vim python config
+.ycm_extra_conf.py
+
+# migrations folder
+migrations/
```

Comment:
Is there a reason we are ignoring migrations? According to the [Django migrations documentation](https://docs.djangoproject.com/en/2.1/topics/migrations/):

`
The migration files for each app live in a “migrations” directory inside of that app, and are designed to be committed to, and distributed as part of, its codebase.
You should be making them once on your development machine and then running the same migrations on your colleagues’ machines, your staging machines, and eventually your production machines.
`

Reply:
Thank you for the correction. I will fix this.